src/components/pages/MainPage.js

```diff
@@ -36,12 +36,12 @@ const MainPage = ({nav}) => {
     const connect = async () => {
         setIsLoading(true);
         try {
-            const desktopUrl = await apiCall(authContext, "connect");
-            nav.go(Pages.CONNECT, {desktopUrl})
+            const desktopUrl = await apiCall(authContext, "connect", {screenHeight: window.innerHeight, screenWidth: window.innerWidth});
```

Comment:
You are sending the size of the current browser window, don't we want send the display size (screen.width and screen.height), since ideally the user will want to maximise their screen. Though since resolutions are fixed, I assume the OS is going to choose an WidthxHeight which closly matches.

Reply:
From the data gathered - it seems users don't tend to maximise. I'm guessing they like to have multiple windows open on screen.

Reply:
Ideally we would resize when they resize the window - but that doesn't seem to be possible right now. This is far from ideal... but seems to be the best available compromise.

Reply:
Once this is tested, I'll look into adding some explanatory text on the connect page warning them to resize before connecting.

Reply:
Not sure I agree, since we want the Chrome desktop to have the same aspect ratio as the screen we are displaying on, we are making an assumption here that the user has already maximised their browser so that the best resolution match is apparent. Why go to the hassle of adding text to the page telling them to resize their browser when you can just send the values directly?

Reply:
The aim of the change was to avoid scaling rather than change the aspect ratio - the problem seems to be the screen being substantially downsampled because we chose a relatively high default resolution.

We suspect this was being exacerbated by different screen aspect ratios... but that wasn't the primary cause.

Reply:
That said - we could choose to calculate the height requested based on the device aspect ratio and the window width?

Reply:
We are passing in variables named screenWidth and screenHeight. Lets pass those actual variables in.